### PR TITLE
changed all name from PhoneAuth to GymFlow

### PR DESCRIPTION
**Pull request by dhwang154**

### CODE REVIEW

**Comment by dhwang154** [comment]
changed PhoneAuth -> GymFlow, and also changed the tab bar to be more seamless with the UI, and fixed ID icon

File: GymFlow.xcodeproj/xcuserdata/danielhwang27.xcuserdatad/xcschemes/Copy of PhoneAuth.xcscheme
Comment (Tommygithubaccount123) [inline GymFlow.xcodeproj/xcuserdata/danielhwang27.xcuserdatad/xcschemes/Copy of PhoneAuth.xcscheme:1]: do not include this file, add `GymFlow.xcodeproj/xcuserdata/*` to gitignore

File: GymFlow.xcworkspace/xcuserdata/danielhwang27.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist
Comment (Tommygithubaccount123) [inline GymFlow.xcworkspace/xcuserdata/danielhwang27.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist:1]: do not include this file, add `GymFlow.xcworkspace/xcuserdata/*` to gitignore

**Review by Tommygithubaccount123 — CHANGES_REQUESTED** [review]

File: PhoneAuth/Info.plist
Comment (Tommygithubaccount123) [inline PhoneAuth/Info.plist:1]: we need to update readme for this missing file now or just keep this file in the branch

**Pull request closed without merging**